dialogue/TownNPCs/Mocha/RandomAzura/Azura_Mocha_Random1.json

```diff
@@ -22,12 +22,20 @@
       "¡No hay nada que yo no sepa!",
       "<w>Ehhh,</w> debería de volver al trabajo."
     ],
+    "pl": [
+      "Ty wiesz, @CharacterName...\nLepiej nie zostawiać swoje drzwi otwarte na zbyt długo.",
+      "Kolorowe gości mogą myszkować i zabrać swoją książkę, w której piszesz dużo informacji.",
```

Comment:
"Kolorowi goście" is gramatically correct, and you should use "twoją" instead of "swoją". Also I think you could change "w której piszesz dużo informacji." to "w której piszesz o wielu informacjach."